tests/conftest.py

```diff
@@ -108,11 +111,13 @@ def trials(experiment_obj, sagemaker_boto_client):
 
 @pytest.fixture
 def experiments(sagemaker_boto_client):
-    experiment_objs = [
-        experiment.Experiment.create(experiment_name=experiment_name,
-                                     sagemaker_boto_client=sagemaker_boto_client)
-        for experiment_name in names()
-    ]
+    experiment_objs = []
+
+    for experiment_name in names():
+        experiment_objs.append(experiment.Experiment.create(experiment_name=experiment_name,
+                                     sagemaker_boto_client=sagemaker_boto_client))
+        time.sleep(1)
```

Comment:
whats the reason for sleeping here? is there a async process here?

Reply:
there was the problem of throttling of requests causing test failures.  I've already bumped up the limits for the sdk account but i wouldnt be surprised if we encountered throttling issue with multiple developers X multiple concurrent test runs (python 3.5, 3.6, 3.7)

also if you look at other tests there are already a lot of sleeps.

for now tests are not hitting throttling issues though.